_analyzers/token-filters/hunspell.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: default
+title: Hunspell
+parent: Token filters
+nav_order: 160
+---
+
+# Hunspell token filter
+
+The `hunspell` token filter is used for stemming and morphological analysis of words in a specific language. This filter leverages Hunspell dictionaries, which are widely used in spell checkers. It works by breaking down words into their root forms (stemming).
+
+The Hunspell dictionary files are automatically loaded at startup from `<OS_PATH_CONF>/hunspell/<locale>` directory. For example `en_GB` locale should have at least one `.aff` file and one or more `.dic` files in directory `<OS_PATH_CONF>/hunspell/en_GB/`.
```

Comment:
I think it would be nice to add an example configuration. "If you use the default configuration, then your directory structure should be the following...". Alternatively, we can add a sentence to the example to the effect of "The following example assumes you have the following directory structure..."